src/manifolds/MetricManifold.jl

```diff
@@ -86,26 +86,26 @@ where $\Gamma_{ijk}$ are the Christoffel symbols of the first kind, and
 $g^{kl}$ is the inverse of the local representation of the metric tensor.
 The dimensions of the resulting multi-dimensional array are ordered $(l,i,j)$.
 """
-function christoffel_symbols_second(M::MetricManifold, x; backend = :default)
-    ginv = inverse_local_metric(M, x)
-    Γ₁ = christoffel_symbols_first(M, x; backend = backend)
+function christoffel_symbols_second(M::MetricManifold, p; backend = :default)
```

Comment:
Can you use `Γ` instead of `\Gamma` above?

Reply:
Of course. I replaced all occurrences accordingly.